src/pages/pageUrls.js

```diff
@@ -322,7 +322,7 @@ module.exports = {
   AnimeZone: {
     match: [
       '*://www.animezone.pl/odcinki/*',
-      '*://www.animezone.pl/odcinek/*'
+      '*://www.animezone.pl/anime/*'
```

Comment:
The old url schema with odcinek still works. It is better to keep the old url in.